plugins/browser-plugin-button-click-tracking/src/util.ts

```diff
@@ -0,0 +1,65 @@
+import { CommonEventProperties } from '@snowplow/tracker-core';
+import { ButtonClickEvent } from './core';
+
+export type AllowList = { allowlist: string[] };
```

Comment:
Could we add API docs here to explain these types? Also make it clear that the allowlist is a list of classes.